src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -1645,6 +1645,9 @@ static bool TryGetPlatformSocketOption(int32_t socketOptionName, int32_t socketO
                 case SocketOptionName_SO_IP_MULTICAST_TTL:
                     *optName = IPV6_MULTICAST_HOPS;
                     return true;
+                case SocketOptionName_SO_IP_TTL:
+                    *optName = IPV6_UNICAST_HOPS;
+                    return true;
```

Comment:
Have you looked to see whether there are any other options we're handling for IPv4 but missing for IPv6?

Reply:
I did look at other options and I did not find anything obvious.